content/hi/docs/tasks/tools/included/verify-kubectl.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "kubectl इंस्टॉल सत्यापित करें"
+description: "kubectl कैसे सत्यापित करें|"
```

Comment:
```suggestion
description: "kubectl कैसे सत्यापित करें।"
```

Reply:
Please note that `|` and `।` are different characters. We should prefer `।` for पूर्ण विराम